### DISCUSSION TITLE
Logging user input

### DISCUSSION THREAD

**Discussion by IainD925** [microsoft/AI-Chat-App-Hack#77] [Ideas]
I think it'd be useful to have a log of user input to help understand what users want from the data and the language they are using in requests.

**Comment by pamelafox** [comment]
Yes, that's a great idea! A developer asked about that recently in the repo issue tracker and I pointed to some resources: https://github.com/Azure-Samples/azure-search-openai-demo/issues/1226
We don't yet have anything built in to the repo.

**Reply by IainD925** [reply]
Excellent, will look to implement.  Thanks.
